scripts/build-upgrade-image.sh

```diff
@@ -93,6 +106,12 @@ cp version.info.template version.info
 VERSION=$(dpkg -f "$(find debs/ -name 'delphix-entire-*' | head -n 1)" version)
 sed -i "s/@@VERSION@@/$VERSION/" version.info
 
+cp verification-version.info.template verification-version.info
+
+# Include version information about this image.
+VERIFICATION_VERSION=$(dpkg -f "$(find debs/ -name 'delphix-upgrade-verification*.deb' | head -n 1)" version)
+sed -i "s/@@VERIFICATION_VERSION@@/$VERIFICATION_VERSION/" verification-version.info
```

Comment:
Can we also put this information into `version.info` as well?

I understand that we have to use the `verification-version.info` file for existing release (i.e. 6.0.0 and 6.0.1), but I think we should adapt future releases to use the `version.info` file, and then we can eventually remove the `verification-version.info` file (once we no longer support upgrading from 6.0.0 and 6.0.1).

With that said, I don't feel strongly; IMO, we should have planned to use `version.info` from the start, but that's a tangent...

Reply:
Added it to version.info file. But it should be noted that we do not expect to be able to remove the verification-version.info file till at least 6.3 - and maybe even later if we start loosening the restrictions on 2 major release upgrade as a maximum.

Reply:
Right, I understand.

This will also allow us to easily retrieve the verification version without unpacking the image. We may not need that functionality now, but it might be useful.. e.g. when unpacking, to determine if we're unpacking an already unpacked version, but with a new/different verification version..